dat/ships/zalek_demon.xml

```diff
@@ -25,10 +25,12 @@
   <cargo>50</cargo>
  </characteristics>
  <slots>
```

Comment:
I have a feeling that this is making it much stronger than it was, but I guess I'll have to run numbers and test. What is the reasoning behind adding two slots and making a medium one large?

Reply:
> I have a feeling that this is making it much stronger than it was, but I guess I'll have to run numbers and test. What is the reasoning behind adding two slots and making a medium one large?

My logic was that it's outfits were worse than the Sting's. Will probably revisit it this when I feel confident enough to add the drone bays to the equip file.